lib/wovnrb.rb

```diff
@@ -86,10 +86,9 @@ def switch_lang(body, values, url, lang=@store.settings['default_lang'], headers
         d.encoding = "UTF-8"
 
         # If this page has wovn-ignore in the html tag, don't do anything
-        if ignore_all || !d.xpath('//html[@wovn-ignore]').empty?
+        if ignore_all || !d.xpath('//html[@wovn-ignore]').empty? || is_amp_page?(d)
           ignore_all = true
-          output = d.to_html.gsub(/href="([^"]*)"/) { |m| "href=\"#{URI.decode($1)}\"" }
```

Comment:
can we remove this line?

Reply:
This line is creating output from html that has not been change and is doing URI fixes.
If content should not be changed then it makes sens to return the original content.

This broke no tests so it seems right?

Reply:
I agree this is not needed.
This change breaks backward compatibility for not only amp users.
So even if there is no test, we need to check there is no affect for users..

Reply:
then i revert